src/fairplay.js

```diff
@@ -0,0 +1,40 @@
+import { stringToArray } from './util';
+
+export default function concatInitDataIdAndCertificate(initData, id, certificate) {
+  if (typeof id === 'string') {
+    id = stringToArray(id);
```

Comment:
In general you don't want to reassign params?

http://eslint.org/docs/rules/no-param-reassign